project/plugins.sbt

```diff
@@ -6,7 +6,7 @@ resolvers ++= Seq(
   Resolver.bintrayIvyRepo("iheartradio", "sbt-plugins")
 )
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.11")
```

Comment:
The update here came after getting the compile error mentioned in this thread [this github thread](https://github.com/sbt/sbt/issues/6400) and following guidance to bump play to >=2.8.8